doc/userguide/setting-up-ipsinline-for-linux.rst

```diff
@@ -1,7 +1,8 @@
 Setting up IPS/inline for Linux
 ================================
 
-In this guide will be explained how to work with Suricata in layer3 inline mode and how to set iptables for that purpose.
+In this guide will be explained how to work with Suricata in layer3
```

Comment:
is there a pro/con of using manual line breaks? I'm not seeing much diff in output.